.gitignore

```diff
@@ -64,3 +64,6 @@ yarn-debug.log*
 /node_modules
 yarn-debug.log*
 .yarn-integrity
+
+# Ignore master key for decrypting credentials and more.
+/config/master.key
```

Comment:
This is also on line 19 of this file, do we need to add it again?

Reply:
Whoops missed that!